charts/ingress-nginx/templates/controller-daemonset.yaml

```diff
@@ -184,6 +184,7 @@ spec:
       {{- end }}
       {{- if .Values.controller.opentelemetry.enabled}}
           {{ $otelContainerSecurityContext := $.Values.controller.opentelemetry.containerSecurityContext | default $.Values.controller.containerSecurityContext }}
+          {{ $fullImage := printf "%s/%s:%s@%s" .Values.controller.opentelemetry.image.registry .Values.controller.opentelemetry.image.image .Values.controller.opentelemetry.image.tag .Values.controller.opentelemetry.image.digest }}
```

Comment:
Same as for https://github.com/kubernetes/ingress-nginx/pull/9773/files#diff-147eaabb2f1187e85b285ac15fc2542b761ce6b8b0ed100102df85b878701575L192-R194.